Make this build by adding <sys/xattr.h> before <sys/capability.h>.

--- Src/Modules/attr.c.orig	2014-01-06 12:26:00.886670913 +0100
+++ Src/Modules/attr.c	2014-01-06 12:26:10.597587472 +0100
@@ -27,12 +27,12 @@
  *
  */
 
-#include "attr.mdh"
-#include "attr.pro"
-
 #include <sys/types.h>
 #include <sys/xattr.h>
 
+#include "attr.mdh"
+#include "attr.pro"
+
 static ssize_t
 xgetxattr(const char *path, const char *name, void *value, size_t size, int symlink)
 {
